internal/mode/static/nginx/config/upstreams.go

```diff
@@ -22,7 +22,7 @@ const (
 	// plusZoneSize is the upstream zone size for nginx plus.
 	plusZoneSize = "1m"
 	// invalidBackendZoneSize is the upstream zone size for the invalid backend upstream.
-	invalidBackendZoneSize = "32k"
+	invalidBackendZoneSize = ""
```

Comment:
We can get rid of this variable since the zero value for a string is `""`.

Reply:
Yea I was considering doing that. If I were to do so, in 
```
func createInvalidBackendRefUpstream() http.Upstream {
	return http.Upstream{
		Name:     invalidBackendRef,
		ZoneSize: invalidBackendZoneSize,
		Servers: []http.UpstreamServer{
			{
				Address: nginx500Server,
			},
		},
	}
}
```
Would you recommend me removing `ZoneSize` entirely? or setting it equal to empty string. 

I was debating if the clarity of having a variable representing the `invalidBackendZoneSize` and purposely setting it to `""` made more sense than removing it, and removing the ZoneSize. 

If I were to do the latter could I add a reason why its removed?
```

Reply:
Ok, removed all of it. 

My worry of not leaving a comment explaining why there is no ZoneSize on the invalidUpstream is that it would be difficult for someone to know why there is no ZoneSize as the answer of 

"Do not specify zone size for invalid-backend-ref so that NGINX does not share upstream state across backends. This is ok for this specific upstream because we will only ever proxy to one backend."

doesn't seem intuitive. But if you think its not necessary then thats alright too.

Reply:
Feel free to add a comment with that explanation in the `createInvalidBackendRefUpstream` function.